pkgs/development/php-packages/snuffleupagus/default.nix

```
@@ -0,0 +1,39 @@
+{ buildPecl, lib, php, fetchFromGitHub, pcre' }:
+
+buildPecl rec {
+  pname = "snuffleupagus";
+  version = "0.7.0";
+  src = fetchFromGitHub {
+    owner = "jvoisin";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1la6wa9xznc110b7isiy502x71mkvhisq6m8llhczpq4rs4nbcw2";
+  };
+
+  buildInputs = [
+    pcre'
+  ];
+
+  internalDeps = with php.extensions; [
+    session
+  ] ++ lib.optionals (lib.versionOlder php.version "7.4") [
+    hash
+  ];
+
+  sourceRoot = "source/src";
+
+  configureFlags = [
+    "--enable-snuffleupagus"
+  ];
+
+  postPhpize = ''
+    ./configure --enable-snuffleupagus
+  '';
+
+  meta = with lib; {
+    description = "Security module for php7 and php8 - Killing bugclasses and virtual-patching the rest!";
+    license = licenses.lgpl3Only;
+    homepage = "https://github.com/jvoisin/snuffleupagus";
+    maintainers = teams.php.members;
```

```suggestion
    maintainers = teams.php.members ++ maintainers.NorfairKing;
```

or whatever is your name.

---

I would prefer not to maintain this. If anyone, it should be `niteoweb`.

---

But you want to use it? Well we don't need to add you but putting the maintenance burden on the php team is not nice and it might just break because no one cares.

---

@SuperSandro2000 Yes that's a fair point, is there a way to add an organisation as a maintainer?

---

Like a company? We have several companies which are listed as teams of maintainers, if that is what you mean. Very welcome, if so.

Is that what you mean?

---

@aanderse. I've added the head of the company: @zupo as the maintainer because he was already in the maintainers list.
It's `niteoweb` that will be using this, I just packaged `snuffleupagus` for them but won't be using it personally.